coral/src/domain/topic/topic-transformer.test.ts

```diff
@@ -1,22 +1,25 @@
-import {
-  transformGetTopicAdvancedConfigOptionsResponse,
-  transformGetTopicRequestsResponse,
-  transformTopicApiResponse,
-  transformTopicOverviewResponse,
-} from "src/domain/topic/topic-transformer";
 import {
   Topic,
   TopicApiResponse,
   TopicRequestApiResponse,
   TopicRequest,
   TopicOverview,
 } from "src/domain/topic/topic-types";
+
+import {
+  transformGetTopicAdvancedConfigOptionsResponse,
+  transformGetTopicRequestsResponse,
+  transformTopicApiResponse,
+  transformTopicOverviewResponse,
+} from "src/domain/topic/topic-transformer";
+import "src/services/test-utils/mock-documenation-helper";
 import {
   baseTestObjectMockedTopic,
   createMockTopicApiResponse,
 } from "src/domain/topic/topic-test-helper";
 import { KlawApiResponse } from "types/utils";
 
+// const mockCreateMarkdown = jest.fn();
```

Comment:
Important or forgotten? ^^

Reply:
Forgotten 🙈  Thanks for always noticing this small things!